policyengine_us/variables/gov/states/mi/tax/income/deductions/retirement/mi_pension_benefit.py

```diff
@@ -20,4 +20,5 @@ class mi_pension_benefit(Variable):
     adds = [
```

Comment:
we might need to make this a formula so it does not break per 2023 

add an bool var that is false in 2022 and true in 2023 and then use an if statement here